### PR TITLE
Tmin_days_below index with class & tests

### PR DESCRIPTION
**Pull request by tlogan2000**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, etc.)
tasmin_days_below index & class

* **Does this PR introduce a breaking change?** (Has there been an API change?)

* **Other information**:
makes use of the recent generic threshold utility

* When merging, please put `fixes #{issue number}` in your comment to auto-close the issue that your PR addresses. Thanks!

### CODE REVIEW

File: xclim/indices.py [hunk 1]
```diff
@@ -958,6 +958,38 @@ def liquid_precip_ratio(pr, prsn=None, tas=None, freq='QS-DEC'):
     return ratio
 
 
+def tn_days_below(tasmin, thresh=-10.0, freq='YS'):
+    r"""Number of days with tmin below a threshold in
```
Comment (huard) [inline xclim/indices.py]: Missing end of sentence (units ?)

File: xclim/indices.py [hunk 2]
```diff
@@ -958,6 +958,38 @@ def liquid_precip_ratio(pr, prsn=None, tas=None, freq='QS-DEC'):
...
+
+    Number of days where daily maximum temperature exceed a threshold.
```
Comment (huard) [inline xclim/indices.py]: not exceed

File: xclim/temperature.py [hunk 3]
```diff
@@ -73,6 +74,15 @@ def compute(*args, **kwds):
         """The function computing the indicator."""
 
 
+tn_days_below = Tasmin(identifier='tnlt_{thresh}',
+                       units='days',
+                       standard_name='number_of_days_with_air_temperature_below_threshold',
+                       long_name='Number of days with Tmin < {thresh}C',
+                       description="{freq} number of days where daily minimum temperature is below {thresh}℃",
+                       cell_methods='time: minimum within days time: sum over days',
```
Comment (huard) [inline xclim/temperature.py]: i don't think it's a case of "sum over days".

Comment (tlogan2000) [reply xclim/temperature.py]: Cell methods on the indicators become very confusing I find but  I took this based on the proposed cf standard excel sheet (from Lars Barring et al)
e.g.  Tropical nights (which is the same but simply greater than) they have:

number_of_days_with_air_temperature_above_threshold | time: minimum   within days time: sum over days | Number of   Tropical Nights (Tmin > 20C)
-- | -- | --

**Pull request merged**